addons/default/visiosoft/base-theme/resources/views/partials/metatags.twig

```diff
@@ -4,7 +4,7 @@
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <meta name="generator" content="{{ config_get('streams::distribution.name') }}"/>
 <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
-<meta name="description" content="{{ trans(template.meta_description) }}"/>
+<meta name="description" content="{{ setting_value('streams::description') }}"/>
```

Comment:
is it working with all pages? @Diatrex 

@spektra2147

Reply:
yes